### DISCUSSION TITLE
10.4kbit speed

### DISCUSSION THREAD

**Discussion by Erklei** [collin80/SavvyCAN#822] [Q&A]
Is it possible to set PeakCAN speed to 10.4kbits, like GMLAN slow uses? I see 33333 as the slowest.

**Comment by mrbartek96** [comment]
In Connection Window what type of connection are you using to add PeakCAN?
From software perispective, the bitrate is given from cbBusSpeed combo box. By default you can only select predefined values, bu if you change property EDITABLE from false to true, you should be able to enter any bitrate as you wish. You should add code to verify that teext entered to cbBusSpeed is convertable to int (don't vae a letters or special chars) In GVRET protocol bitrate is sended to device as uint.
